server/cmake/modules/FindLibClang.cmake

```diff
@@ -16,7 +16,7 @@
 # most recent versions come first
 # http://llvm.org/apt/
 set(LIBCLANG_KNOWN_LLVM_VERSIONS 5.0.0 5.0
```

Comment:
path is `/usr/lib/llvm/4/lib64/cmake/clang/ClangConfig.cmake`

[ClangConfig.cmake.txt](https://github.com/Sarcasm/irony-mode/files/862702/ClangConfig.cmake.txt)